modules/react/button/stories/visual-testing/stories_DeleteButton.tsx

```diff
@@ -19,10 +20,20 @@ export const DeleteButtonStates = (props: {theme?: PartialEmotionCanvasTheme}) =
     <ComponentStatesTable
       rowProps={permutateProps({
         size: [
+          {value: 'extraSmall', label: 'Extra Small'},
           {value: 'small', label: 'Small'},
           {value: 'medium', label: 'Medium'},
           {value: 'large', label: 'Large'},
         ],
+        icon: [
+          {value: undefined, label: ''},
+          {value: trashIcon, label: ''},
+        ],
+        iconPosition: [
+          {value: undefined, label: ''},
+          {value: 'start', label: '& Left Icon'},
+          {value: 'end', label: '& Right Icon'},
+        ],
       })}
```

Comment:
I see visual tests have some unrelated lines with buttons, because prop states are not filtered. You also need to add filter function as a second argument in `permutateProps` like here:

https://github.com/Workday/canvas-kit/blob/9c4f6d3300801be8aca7be5b616ce6520ab7f7f8/modules/react/button/stories/visual-testing/stories_PrimaryButton.tsx#L44-L45

Reply:
Oh I see.